src/shared/containers/AdminPluginContainer.tsx

```diff
@@ -71,23 +71,6 @@ const AdminPlugin: React.FunctionComponent<AdminProps> = ({
 
   const content = (
     <>
-      <AccessControl
-        path={`/${orgLabel}/${projectLabel}`}
-        permissions={['resources/write']}
-        noAccessComponent={() => <></>}
-      >
-        {editable && (
-          <Alert
-            message={
-              <>
-                <EditOutlined /> You can edit this resource.
-              </>
-            }
-            type="success"
-          />
-        )}
-      </AccessControl>
-
```

Comment:
is there is a reason why you remove the check for permission for this one

Reply:
`I removed the "You can edit this resource" information as I actually removed this in a PR before the migration`. We once talked about this if I could remove this some months ago, and it came back after the migration rebase => so I removed it again. I can bring it back if you think it's still necessary.